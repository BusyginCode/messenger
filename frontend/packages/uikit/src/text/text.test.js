import React from 'react'
import { render } from 'enzyme'
import Text from './text'

describe('uikit/Text', () => {
  it('should renders correctly', () => {
    const wrapper = render(<Text>Text</Text>)
    expect(wrapper).toMatchSnapshot()
  })

  it('should renders correctly with inject', () => {
    const wrapper = render(
      <Text inject>
        <a />
      </Text>
    )
    expect(wrapper).toMatchSnapshot()
  })
})
