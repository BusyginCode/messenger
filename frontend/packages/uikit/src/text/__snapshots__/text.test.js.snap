// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`uikit/Text should renders correctly 1`] = `
<p
  class="base general m left"
>
  Text
</p>
`;

exports[`uikit/Text should renders correctly with inject 1`] = `
<a
  class="base general m left"
/>
`;
